src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+    class="search__bar big__bar"
+    method="post"
+    data-qa="big">
+      <div class="biginput">
+        <label for="biginput"
+        class="search__icon icon--big"> <img src="./images/Search.svg" alt="search">
+        </label>
+        <input
+        class="search__input"
+        type="text"
+        name="biginput"
+        data-qa="keypress"
+      placeholder="Try “Los Angeles”">
+      </div>
+    </form>
+
+    <form
+    class="search__bar search__bar--small"
+    method="post"
+    data-qa="small">
+      <div class="smallinput">
+        <label
+        for="biginput"
+        class="search__icon icon--small"> <img src="./images/Search.svg" alt="search">
+        </label>
+        <input
+        class="search__input search__input--small"
+        type="text"
+        name="smallinput"
+        placeholder="Try &quot;Los Angeles&quot;">
+      </div>
+    </form>
```

Comment:
```suggestion
    <form
      class="search__bar big__bar"
      method="post"
      data-qa="big"
    >
      <div class="biginput">
        <label 
          for="biginput"
          class="search__icon icon--big"
        >
          <img src="./images/Search.svg" alt="search">
        </label>
        <input
          class="search__input"
          type="text"
          name="biginput"
          data-qa="keypress"
          placeholder="Try “Los Angeles”"
        >
      </div>
    </form>

    <form
      class="search__bar search__bar--small"
      method="post"
      data-qa="small"
    >
      <div class="smallinput">
        <label
          for="biginput"
          class="search__icon icon--small"
        >
          <img src="./images/Search.svg" alt="search">
        </label>
        <input
          class="search__input search__input--small"
          type="text"
          name="smallinput"
          placeholder="Try &quot;Los Angeles&quot;"
        >
      </div>
    </form>
```
Consider fixing indents according to the code style guide